CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 0.21.0 (August 1, 2023)
```

Comment:
what part of this change is breaking? is the  `opentelemetry_sdk` dependency switch a breaking change, or are the other changes API-breaking? can we document why this is a breaking change version?

Reply:
Looks like #43 added a type argument to the public `MetricsLayer` type, at least.

Reply:
I don't feel strongly about it, we could consider this a patch release instead for easier upgrades

Reply:
I suppose the question is, how likely is the extra type argument to break downstream users? But if we don't want to do that analysis, safer to just eat the minor version bump.

Reply:
The added type argument is a breaking change, since it means users who have written e.g. functions that return a `MetricsLayer` will find that their code no longer compiles. I think it would be incorrect to release that change in a patch version.

I'd like it if the changelog could explicitly state what's breaking here, in addition to listing these changes. Can we do that?